capstone/fabfile.py

```diff
@@ -1147,6 +1148,17 @@ def extract_all_citations(last_run_before=None):
     """ extract citations """
     tasks.run_task_for_volumes(tasks.extract_citations_per_vol, last_run_before=last_run_before)
 
+    # concatenate all citations into one big file
+    def concat_and_remove_file(smallfile):
+        with open("/tmp/missed_citations.csv", "a+") as extracted_citations_file:
+            writer = csv.writer(extracted_citations_file)
+            with open(smallfile) as f:
+                reader = csv.reader(f)
+                [writer.writerow(row) for row in reader]
+            os.remove(smallfile)
+
+    [concat_and_remove_file(f) for f in glob('/tmp/missed_citations-*.csv')]
+
```

Comment:
This doesn't make sense to do here, as it will run before any of the volumes are processed. Maybe we just do it as `cat /tmp/missed_citations-*.csv >> missed_citations.csv` after the tasks are complete?